File: python/7 kyu/227.py
# https://www.codewars.com/kata/55caef80d691f65cb6000040/train/python

# Passed

def geometric_sequence_elements(a, r, n):
    sequence = [str(a * r ** pow) for pow in range(n)]
    sequence_str = ", ".join(sequence)
    return sequence_str

output = geometric_sequence_elements(2, 3, 5)
print(output)

File: nodejs/8 kyu/227.js
// https://www.codewars.com/kata/5b4e779c578c6a898e0005c5/train/javascript

// Passed

function drawStairs(n) {
  let rows = [];
  for (let rowCount = 1; rowCount <= n; rowCount++) {
    let row = " ".repeat(rowCount - 1) + "I";
    rows.push(row);
  }

  let stairs = rows.join("\n");
  return stairs;
}

const output = drawStairs(5);
console.log(output);